docs: add log level docs for pg_audit (#14421)
supabase/supabase: apps/docs/pages/guides/platform/logs.mdx

alter role postgres set pgaudit.log to 'function, write, ddl';
```

> It is advised to adjust the scope of the logs to only relevant statements to aid in your debugging, as having too wide of a scope would result in a lot of noise in your Postgres logs.

Reviewer: ```suggestion
To help with debugging, we recommend adjusting the log scope to only relevant statements as having too wide of a scope would result in a lot of noise in your Postgres logs.
```
